ops-bedrock/Dockerfile.l2

```diff
@@ -1,4 +1,4 @@
-FROM ethereumoptimism/op-geth:optimism
+FROM us-docker.pkg.dev/oplabs-tools-artifacts/images/op-geth:optimism
```

Comment:
Is this change to the dockerfile something that we will be planning on keeping?

Reply:
yea, we're trying to migrate to `us-docker.pkg.dev` from dockerhub